.github/workflows/psalm.yml

```diff
@@ -19,12 +19,31 @@ jobs:
     name: Psalm
     runs-on: Ubuntu-20.04
 
+    env:
+      php-version: '8.1'
+      extensions: json,couchbase,memcached,mongodb,redis,xsl,ldap,dom
+      key: cache-v1 # can be any string, change to clear the extension cache.
     steps:
+      - name: Setup cache environment
+        id: extcache
+        uses: shivammathur/cache-extensions@v1
+        with:
+          php-version: ${{ env.php-version }}
+          extensions: ${{ env.extensions }}
+          key: ${{ env.key }}
+
+      - name: Cache extensions
```

Comment:
This name (_"Cache extensions"_) seems more appropriate for the previous step (`uses: shivammathur/cache-extensions`) instead of this step (`uses: actions/cache`)

Reply:
I took these labels from [the example workflow](https://github.com/shivammathur/cache-extensions#workflow). I agree it's a little confusing on first sight, but I'm not sure what would be a better label. Both steps are responsible for caching in their own way.

Reply:
OK, let's keep it that way then. Thanks for checking this.